docs/index.yaml

```diff
@@ -3,7 +3,7 @@ entries:
   andi:
   - apiVersion: v1
     appVersion: "1.0"
-    created: 2019-07-17T08:43:03.913273417-04:00
+    created: "2019-07-17T08:43:03.913273417-04:00"
```

Comment:
Did you run `package.sh`? I think @smillardHNTB fixed this issue where the created field was overwritten for each package.